molsysmt/item/file_mmtf/to_mdtraj.py

```diff
@@ -132,8 +132,7 @@ def __init__(self, filename, mode='r', force_overwrite=True):
             self._open = False
             self._frame_index = 0
             self._filepath = filename
-            try:
-                self.n_atoms, self.topology = self._read_topology()
+            self.n_atoms, self.topology = self._read_topology()
```

Comment:
This try is because sometimes there is no _read_topology or it fails... and in this case n_atoms and topology will be empty. But the method needs to continue.

Reply:
This `try` was not followed by an `except` or a `finally`. Trying to import the file caused a syntax error, so I removed it.

Reply:
Ok. Let me check it because this code has been imported many times without any error. What version of Python are you using? Is there a reason explaining why I had no errors?

Reply:
A revision is necessary. Reported in #65